web/modules/custom/mymodule/mymodule.module

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * This File Demonstrates how to use the hook functions.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements the hook_cron() function.
+ */
+function mymodule_cron() {
+  Drupal::messenger()->addMessage(t('Hello This is Our custom module'));
```

Comment:
Use \Drupal::service('messenger') instead of using the above.

Reply:
@Eshakundu01 can you tell me the reason why we should use \Drupal::service() instead of the above?

Reply:
I was trying to say that we should use \Drupal as well as I had read somewhere we can use services with \Drupal::service() in the module file and as @Vishalprasad28 has implemented dependency injection so I suggested.

Reply:
Actually both ways are correct for now.
So no need to change. :)

Reply:
Ok